static/templates/settings/deactivation_user_modal.hbs

```diff
@@ -1,19 +1,6 @@
-<div id="deactivation_user_modal" class="modal modal-bg hide fade" tabindex="-1" role="dialog" aria-labelledby="deactivation_user_modal_label" aria-hidden="true">
-    <div class="modal-header">
-        <button type="button" class="close" data-dismiss="modal" aria-label="{{t 'Close' }}"><span aria-hidden="true">&times;</span></button>
-        <h3 id="deactivation_user_modal_label">{{t "Deactivate" }} <span class="email"></span></h3>
-    </div>
-    <div class="modal-body">
-        <p>
-            {{#tr}}
-                By deactivating <z-user></z-user>, they will be logged out immediately.
-                {{#*inline "z-user"}}<strong><span class="user_name"></span></strong> &lt;<span class="email"></span>&gt;{{/inline}}
-            {{/tr}}
-        </p>
-        <p>{{t "Their password will be cleared from our systems, and any bots they maintain will be disabled." }}</p>
-    </div>
-    <div class="modal-footer">
-        <button class="button rounded" data-dismiss="modal">{{t "Cancel" }}</button>
-        <button class="button rounded btn-danger do_deactivate_button">{{t "Confirm" }}</button>
-    </div>
-</div>
+<p>
+    {{#tr}}
+    By deactivating <strong>{{username}}</strong> &lt;{{email}}&gt;, they will be logged out immediately.
```

Comment:
The `#*inline` `z-user` syntax is important for the translation experience (and is why you're getting CI failures).  

Please preserve that formatting of the translated strings.